ci/github-script/bot.js

```diff
@@ -212,13 +212,14 @@ module.exports = async ({ github, context, core, dry }) => {
 
     // Check for any human reviews other than GitHub actions and other GitHub apps.
     // Accounts could be deleted as well, so don't count them.
+    // And ignore reviews from the author, as they can never approve the PR.
     const reviews = (
       await github.paginate(github.rest.pulls.listReviews, {
         ...context.repo,
         pull_number,
       })
     ).filter(
-      (r) => r.user && !r.user.login.endsWith('[bot]') && r.user.type !== 'Bot',
+      (r) => r.user && !r.user.login.endsWith('[bot]') && r.user.type !== 'Bot' && r.user.login !== pull_request.user?.login,
```

Comment:
Would it be more robust to compare `id` instead of `login`?

It shouldn't usually matter, but there's the unlikely edge case where an account is renamed inbetween getting the `pull_request` payload and the `listReviews` request.